packages/generators/init-template/default/webpack.configjs.tpl

```diff
@@ -68,3 +63,16 @@ module.exports = {
         extensions: ['.tsx', '.ts', '.js'],
     },<% } %>
 };
+
+module.exports = () => {
+    if (process.env.NODE_ENV == 'production') {
+        config.mode = 'production';
+    } else {
+        config.mode = 'development';<% if (devServer) { %>
+        config.devServer = {
+            open: true,
+            host: 'localhost',
+        };<% } %>
+    }
+    return config;
+};
```

Comment:
hm, I think some developers use serve for development and production mode (for testing and etc), so I think keeping `devServer` is good as part of basic configuration, because we use `devServer` option only in `serve` mode

Reply:
Okay, on it. Thanks.

Reply:
Done. Will push snaps (currently running locally) then we can merge.